src/constants/PartnersList.ts

```diff
@@ -0,0 +1,15 @@
+import nestle from "../assets/nestle.svg";
+import apple from "../assets/apple.svg";
+import target from "../assets/target.svg";
+import amazon from "../assets/amazon.svg";
+import google from "../assets/google.svg";
+import walmart from "../assets/walmart.svg";
```

Comment:
What can we do, so we wouldn't need to write paths like this '../../../../'. hint: baseURL